pkg/controller/restore_controller.go

```diff
@@ -488,7 +488,9 @@ func (c *restoreController) runRestore(
 	actions []restore.ItemAction,
 	info backupInfo,
 	pluginManager plugin.Manager,
-) (restoreWarnings, restoreErrors api.RestoreResult, restoreFailure error) {
+) (api.RestoreResult, api.RestoreResult, error) {
```

Comment:
@skriss @carlisia we don't have to do this here, but I dislike having functions that return multiple things of the same type (warnings vs errors for `api.RestoreResult`, in this case). It can lead to confusion and programmer error if you accidentally mess up the order. We should explore alternatives.

Reply:
Agree. An alternative that I often use is to create a struct.

Reply:
exactly :-)